<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_7">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Друзья пользователя</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Get next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Друзья:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_2"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Get next</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Друзья:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_3">
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Общие друзья двух пользователей:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_4">
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Get Mutal</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Список всех совпавших:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_5">
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QWebView" name="webView_2">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>100</height>
          </size>
         </property>
         <property name="url">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QLineEdit" name="lineEdit"/>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Send message</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWebView" name="webView">
     <property name="url">
      <url>
       <string>about:blank</string>
      </url>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>webView</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>listWidget</zorder>
  <zorder>listWidget_2</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>listWidget_3</zorder>
  <zorder>listWidget_4</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>listWidget_5</zorder>
  <zorder>label_6</zorder>
  <zorder></zorder>
  <zorder>listWidget</zorder>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
